rubrik_cdm/data_management.py

```diff
@@ -1620,3 +1620,60 @@ def vcenter_refresh_vm(self, vm_name, timeout=15):  # pylint: ignore
         config = {'vmMoid':moid}
         self.log("vcenter_refresh_vm: Refreshing vSphere VM {} metadata.".format(vm_name))
         self.post('internal', '/vmware/vcenter/{}/refresh_vm'.format(vcenter_id), config, timeout)
+
+    def get_vsphere_vm(self, timeout=15, **query):  # pylint: ignore
```

Comment:
How does `**query` come into play here? Similar to the rest of the Keyword Arguments?

Reply:
Can we use **kwargs to accept keyword arguments and validate the allowed keys within the function? 
Alternative is to define them individually